include/http/middleware/controller.js

```diff
@@ -1,40 +1,53 @@
-const util = require('util')
-const _ = require('lodash')
+const util = require('util');
+const _ = require('lodash');
 module.exports = pb => ({
-    instantiateController: (req, res) => {
-        const Controller = _.get(req, 'route.controller');
-        req.controllerInstance = new Controller();
+    instantiateController: async (ctx, next) => {
+        const Controller = _.get(ctx.req, 'route.controller');
+        ctx.req.controllerInstance = new Controller();
+        await next();
     },
-    initializeController: (req, res) => {
-        const props = pb.RequestHandler.buildControllerContext(req, res, {});
-        return new Promise((resolve, reject) => {
-            req.controllerInstance.init(props, (err) => {
+    initializeController: async (ctx, next) => {
+        const props = pb.RequestHandler.buildControllerContext(ctx);
+        await new Promise((resolve, reject) => {
+            ctx.req.controllerInstance.init(props, (err) => {
                 if (util.isError(err)) {
                     return reject(err)
```

Comment:
Minor: add semicolons to these lines to follow the pattern

Reply:
👍